.azure-pipelines/hpu-tests.yml

```diff
@@ -0,0 +1,59 @@
+# Pipeline to run the HPU tests in DL1 Instance
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - "master"
+      - "release/*"
+      - "refs/tags/*"
+pr:
+  - "master"
+  - "release/*"
+
+jobs:
+  - job: HpuTests
+
+    # how long to run the job before automatically cancelling
+    timeoutInMinutes: "5"
+    # how much time to give 'run always even if cancelled tasks' before stopping them
+    cancelTimeoutInMinutes: "2"
+
+    pool: intel-hpus
+
+    workspace:
+      clean: all
+
+    steps:
+
+    - bash: |
+        echo "Installing pytorch_lightning and required dependencies"
+        python setup.py bdist_wheel
```

Comment:
```suggestion
```
this is not needed

Reply:
we need this to install pytorch lightning wheel